### DISCUSSION TITLE
Working to Bring More Women and People of Color Into the API Specification Conversation

### DISCUSSION THREAD

**Discussion by kinlane** [json-schema-org/community#50] [General]
I wanted to start a conversation around the troubling fact that there are too few women and people of color involved at all levels of this community. AsyncAPI’s Senior Technical Writer Alejandra Quetzall brought it up in the SIG call today, but it is something that continues to get brought up, yet continues to be a problem for us—that the API specification conversation is largely dominated by white men. I know that many of us dread having this conversation, but trust me, it is no fun being the person to bring it up (or be affected by it), and I’d rather do the work than require women and people of color to keep doing the hard work here. So I wanted to kick off this conversation around what we can do to address this issue—I am looking for discussion around two things:

- What obstacles currently keep women from participating in conversations within this community?
- What can we do to attract more women to these communities and empower them to participate?

If you aren’t comfortable having this discussion here on the forum, feel free to email your ideas at kinlane@gmail.com, but I would prefer that we have this conversation out in the open, no matter how uncomfortable it makes us. I’d also like to make sure we keep this conversation friendly, inclusive, and supportive, but also make sure it is an honest and straightforward one. I look forward to hear your ideas and thoughts, and if you are someone from any of the underrepresented communities that are regularly absent from technology and the world of APIs, feel free to ping me directly for help learning about how you can get more involved.

**Comment by jviotti** [comment]
In my previous company, we would reach out to communities representing minorities for i.e. hiring purposes in order to solve this problem. These were either ad-hoc developer groups organized around a certain region, university groups, etc. 

While JSON Schema is not hiring (yet?), I wonder if we could explore giving some presentations about JSON Schema, how it works, what it goes, etc to underrepresented audiences (either in person or online) and encouraging them to continue getting involved with the project, offering mentorship on contributing to JSON Schema in different ways, etc. What do you think?
